### PR TITLE
Adds Migration Example

### PR DESCRIPTION
**Pull request by pmrgnogueira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -465,7 +465,7 @@ Rules to follow when organizing Components (`scss folder`).
 - When a component has dependencies of another components, it needs to be on its own folder, under its parent.
 - The main style file should be the component's name. This file is imported by `main.scss`. This should import all dependencies files. _(check navbar example)_.
 - If the component is compatible with global modifiers, its scss needs to be placed on a separate file named `global-modifiers.scss`.
-- All dependencies file should be prepended with the component's name followed by `-dependencyName`.  
+- All dependencies file should be prepended with the component's name followed by `-dependencyName`.
```
Comment (joaosaro) [inline README.md]: There won't be dependencies. Can we remove all references to that? 🙏

File: README.md [hunk 2]
```diff
@@ -486,3 +486,133 @@ CHANGELOG.md
...
+```scss
+.co-button {
+  .co--small {
+    // Css properties
+  }
+
+  .co--large {
+    // Css properties
+  }
+}
```
Comment (joaosaro) [inline README.md:555]: There will be the concept of `modifiers` and `global-modifiers`? Will we have a styler? Or can we just have component modifiers? 💭 
(cc: Team . I will share in #kobold)

File: README.md [hunk 3]
```diff
@@ -486,3 +485,133 @@ CHANGELOG.md
...
+  .co-button--loader {
+    // Css properties
+  }
+}
+```
+
+Example of a global modifiers file:
+```scss
+.co-button {
+  .co--small {
```
Comment (mmelo) [inline README.md:548]:
```suggestion
  &.co--small {
```
and the same on all other please, or else you are referencing a child and not an actual modiffier..

And I'd even add that the way I'd prefer to add such a modiffier is:
```scss
.co-button {
  &--smal { }
}
```

But with our guidelines should become:

```scss
.co-button {
  &.co-button--smal { }
}
```

File: README.md [hunk 4]
```diff
@@ -486,3 +485,133 @@ CHANGELOG.md
...
+## White Labeling
+
+All components must be compatible with white labeling. To make this possible, some key css properties must use css variables. Each component has different needs so the documentation will provide the correct list of properties that will need to be configurable.
+For the example of the button, the things that may change are the colors and the shape.
+
+How to declare a variable:
+
+```scss
+--color-varible: #{palette-color('gray', 300)};
+--padding-variable: 0 24px;
```
Comment (joaosaro) [inline README.md:590]: CSS vars needs to be nested under a selector (at least `:root`).

Comment (mmelo) [reply README.md:590]: Correct! But this is just a snippet of and does not represent the whole scope of a file or selector, are you suggesting adding one?

**Pull request closed without merging**